pages/supervision/index.tsx

```diff
@@ -1,20 +1,270 @@
+          {Boolean(emailConseillerActuel.error) && (
+            <div className='flex mt-4'>
+              <ImportantIcon
+                role='img'
+                focusable={false}
+                aria-hidden={true}
+                className='fill-status_warning w-6 h-6 mr-2'
+              />
+              <span className=' text-status_warning'>
+                {emailConseillerActuel.error}
+              </span>
+            </div>
+          )}
+        </form>
+
+        <form id='affecter-jeunes' onSubmit={() => {}} className='grow mr-16'>
+          <label
+            htmlFor='email-conseiller-destination'
+            className={`text-base-medium ${
+              areSomeJeunesSelected ? 'text-neutral_content' : 'text-[#999BB3]'
+            }`}
+          >
+            E-mail conseiller de destination
+          </label>
+          <div className='flex mt-3.5'>
+            <>
+              <input
+                type='email'
+                id='email-conseiller-destination'
+                name='email-conseiller-destination'
+                onChange={() => {}}
+                className='flex-1 p-3 w-8/12 border border-r-0 border-neutral_grey rounded-l-medium text-base-medium text-primary_primary disabled:border-[#999BB3]'
+                disabled={!areSomeJeunesSelected}
+              />
+              <button
+                type='reset'
+                title='Effacer'
+                aria-label='Effacer le champ de saisie'
+                className='border border border-l-0 border-content_color rounded-r-medium w-8 text-primary_primary disabled:border-[#999BB3]'
+                onClick={() => {}}
+                disabled={!areSomeJeunesSelected}
+              >
+                <CloseIcon
+                  role='img'
+                  className={`text-primary_primary ${
+                    !areSomeJeunesSelected ? 'fill-[#999BB3]' : ''
+                  }`}
+                  focusable={false}
+                  aria-hidden={true}
+                />
+              </button>
+            </>
+          </div>
+        </form>
+
+        <Button
+          form='affecter-jeunes'
+          label='Réaffecter les jeunes'
+          type='submit'
+          className='mt-9'
+          disabled={!areSomeJeunesSelected}
+        >
+          <ArrowIcon
+            className='fill-blanc mr-2'
+            role='img'
+            focusable='false'
+            aria-hidden={true}
+            width='16px'
+            height='16px'
+          />
+          Réaffecter les jeunes
+        </Button>
+      </div>
+
+      {isRechercheSubmitted && (
+        <div className='mt-6 ml-5'>
+          <span className='text-m-medium'>
+            Jeunes de {emailConseillerActuel.value}
+          </span>
+          <table className='w-full mt-8'>
+            <thead>
+              <tr>
+                <th className='pb-2' />
+                <th className='pb-2 pl-4 pr-4 text-sm-regular text-neutral_content'>
```

Comment:
```suggestion
                <th scope ="col" className='pb-2 pl-4 pr-4 text-sm-regular text-neutral_content'>
```
ajouter cet attribut sur les 3 colonnes `<th>`